core/src/main/kotlin/api/RepositoriesRoute.kt

```diff
@@ -110,8 +110,12 @@ fun Route.repositories() = route("repositories/{repositoryId}") {
             val repositoryId = call.requireIdParameter("repositoryId")
             val pagingOptions = call.pagingOptions(SortProperty("index", SortDirection.ASCENDING))
 
-            val jobsForOrtRuns = repositoryService.getOrtRuns(repositoryId, pagingOptions.mapToModel())
-                .map { it.mapToApiSummary(repositoryService.getJobs(repositoryId, it.index)!!.mapToApiSummary()) }
+            val jobsForOrtRuns = repositoryService.getOrtRuns(repositoryId, pagingOptions.mapToModel()).mapNotNull {
+                repositoryService.getJobs(repositoryId, it.index)?.let { jobs ->
+                    it.mapToApiSummary(jobs.mapToApiSummary())
+                }
```

Comment:
Better use an empty `JobSummaries` object as fallback instead of `mapNotNull`, then at least the run is still in the list even if there is an issue with loading the jobs.

Reply:
IIUC `JobSummaries` does not work here, did you mean a `OrtRunSummary`? But the latter cannot be easily created empty.

Reply:
Ok, I now got what you mean, I believe.